Pipeline/PackageBuildOutputs/PackageBuildOutputs.groovy

```diff
@@ -819,6 +860,19 @@ def parseInput(String[] cliArgs){
 			rc = 2
 		}
 	}
+
+		// assess required options to generate Concert Build manifest
+	if (props.generateConcertBuildManifest && props.generateConcertBuildManifest.toBoolean()) {
+		if (!props.branch) {
+			println("*! [ERROR] Missing branch parameter ('--branch'). It is required for generating the Concert Build Manifest file.")
+			rc = 2
+		}
+		if (!props.publish) {
+			println("*! [ERROR] Missing publish parameter ('--publish'). It is required for generating the Concert Build Manifest file.")
+			rc = 2
+		}
```

Comment:
Is it a correct assumption that generating the concert build manifest file, will only work when only one BuildReport.json is passed to the PackageBuildOutputsScript? If so, we should validate that in this block.

Reply:
We will need to discuss this

Reply:
done. Added the check. Need some more tests